lstchain/reco/r0_to_dl1.py

```diff
@@ -173,14 +173,10 @@ def get_dl1(
             signal_pixels[island_labels != max_island_label] = False
 
         if delta_time is not None:
-            # makes sure only signal pixels are used in the time check:
-            cleaned_pixel_times = peak_time.copy()
-            cleaned_pixel_times[~signal_pixels] = np.nan
-
             signal_pixels = apply_time_delta_cleaning(
                 camera_geometry,
                 signal_pixels,
-                cleaned_pixel_times,
+                peak_time,
```

Comment:
why changing the input from the masked pixel times after cleaning to the original ones?

Reply:
Because that was done specifically to fix a bug in the ctapipe 0.10.5 version